### PR TITLE
Preslist add enddate

### PR DESCRIPTION
**Pull request by sbillinge**

### CODE REVIEW

File: regolith/builders/preslistbuilder.py [hunk 1]
```diff
@@ -166,13 +166,23 @@ def latex(self):
                         2: "$^\mathrm{nd}$",
                         3: "$^\mathrm{rd}$",
                     }.get(pres["begin_day"] % 10, "$^\mathrm{th}$")
+                    for day in ['begin_day', 'end_day']:
+                        if pres.get(day,None):
+                            if 10 < pres[day] < 20:
```
Comment (CJ-Wright) [inline regolith/builders/preslistbuilder.py]: This is pretty slick! Awesome! (we might consider making this into a function and putting it into tools?)

**Comment by CJ-Wright** [comment]
Would it be possible to write a test for this?

**Comment by sbillinge** [comment]
sure.  I would like to pull this into tools as it could be useful elsewhere
I feel, then write a test?  I still need help with tests, but I can have a
first crack at it.

I am pretty happy with how these presentation lists are looking now!

**Comment by CJ-Wright** [comment]
Great! The xonsh PR should be ready to go now, so we'll be able to test if it does the correct thing on windows soon.

**Comment by sbillinge** [comment]
Great

**Comment by sbillinge** [comment]
@CJ-Wright @scopatz this is ready for review.  Thanks!

File: news/preslist_add_enddate.rst [hunk 2]
```diff
@@ -0,0 +1,17 @@
+**Added:**
+
+* number_suffix function to tools, returns the suffice to turn numbers into adjectives
```
Comment (CJ-Wright) [inline news/preslist_add_enddate.rst]: "returns the suffix"?

File: regolith/tools.py [hunk 3]
```diff
@@ -400,10 +400,35 @@ def fuzzy_retrieval(documents, sources, value, case_sensitive = True):
...
+    ---------
+    number: integer
+        The number.  If number is not an integer, returns an empty string
+
+    Returns
+    -------
+    suffix: string
+        The suffix (st, nd, rd, th)
+    """
+    if not isinstance(number, (int, float)):
```
Comment (CJ-Wright) [inline regolith/tools.py]: Do we actually allow float days?

Comment (scopatz) [reply regolith/tools.py]: Doesn't seem to hurt really/

**Comment by scopatz** [comment]
Thanks @sbillinge!

**Comment by sbillinge** [comment]
thanks @scopatz

**Pull request merged**